src/components/features/PostForm.js

```diff
@@ -1,69 +1,88 @@
 import { Form, Button } from 'react-bootstrap';
 import { useState } from 'react';
-//import clsx from 'clsx';
-import FormText from '../views/FormText';
-import FormTextArea from '../views/FormTextArea';
+import { useForm } from "react-hook-form";
+//import FormText from '../views/FormText';
+//import FormTextArea from '../views/FormTextArea';
+import ReactQuill from 'react-quill';
+import 'react-quill/dist/quill.snow.css';
+import DatePicker from "react-datepicker";
+import "react-datepicker/dist/react-datepicker.css";
+import clsx from 'clsx';
+
+
 
 
 const PostForm = ({ action, actionText, ...props }) => {
 
+  const { register, handleSubmit: validate, formState: { errors } } = useForm();
   const [title, setTitle] = useState(props.title || '');
   const [author, setAuthor] = useState(props.author || '');
-  const [publishedDate, setPublishedDate] = useState(props.publishedDate || '');
+  const [publishedDate, setPublishedDate] = useState(props.publishedDate || new Date());
   const [shortDescription, setShortDescription] = useState(props.shortDescription || '');
   const [content, setContent] = useState(props.content || '');
+  const [contentError, setContentError] = useState(false);
+  const [dateError, setDateError] = useState(false);
+
+
 
-  const handleSubmit = e => {
-    e.preventDefault();
-    action({ title, author, publishedDate, shortDescription, content });
-  }
+  const handleSubmit = () => {
+    setContentError(!content)
+    setDateError(!publishedDate)
+    if(content && publishedDate) {
+      action({ title, author, publishedDate, shortDescription, content });
+    }
+  };
 
 
   return (
-    <Form onSubmit={handleSubmit} className={styles.form}>
-      <FormText
-        controlId="formTitle"
-        label="Title"
-        placeholder="Enter title"
-        value={title}
-        action={setTitle}
-      />
-      <FormText
-        controlId="formAuthor"
-        label="Author"
-        placeholder="Enter author"
-        value={author}
-        action={setAuthor}
-        className={styles.smallInput}
-      />
-      <FormText
-        controlId="formDate"
-        label="Published"
-        placeholder="Enter date"
-        value={publishedDate}
-        action={setPublishedDate}
-        className={styles.smallInput}
-      />
-      <FormTextArea
-        controlId="formDescription"
-        label="Short description"
-        placeholder="Enter description"
-        value={shortDescription}
-        action={setShortDescription}
-        rows={3}
-      />
-      <FormTextArea
-        controlId="formContent"
-        label="Main content"
-        placeholder="Enter content"
-        value={content}
-        action={setContent}
-        rows={10}
-      />
-      <Button variant="primary" type="submit">{actionText}</Button>
-    </Form>
-  )
-}
+        <Form onSubmit={validate(handleSubmit)} className={styles.form}>
+          <Form.Group className={clsx("mb-3")} controlId="formTitle">
```

Comment:
Jeżeli nie przekazujesz żadnych warunkowych nazw klas, to nie ma potrzeby stosowania clsx